src/clip.sol

```diff
@@ -91,14 +89,19 @@ contract Clipper is LibNote {
     uint256 public stopped = 0;
 
     // --- Events ---
+    event Rely(address indexed usr);
+    event Deny(address indexed usr);
+
+    event File(bytes32 indexed what, uint256 data);
+    event File(bytes32 indexed what, address data);
```

Comment:
I might actually name these differently to prevent confusion, e.g. `FileUint` and `FileAddress`.

Reply:
Ahh sorry just saw this. I actually prefer both with `File` name, but not opposed if you guys prefer to differentiate them.

Reply:
I don't feel incredibly strongly either, just thought it might be easier for human eyes when looking at Etherscan or something to lessen the chance of a mistake. I can live with it either way.